antora-ui-camel/src/css/header.css

```
@@ -356,7 +418,7 @@ body {
   }
 
   #search-cancel {
-    bottom: calc(50% - 0.45rem);
+    margin-top: 0.75rem;
   }
 
   #search_results {
```

I think it looks a bit better if we take up full width on smaller screens
```suggestion
  #search_results {
    left: 0;
    width: 100vw;
```